doc/api/thalireplicationmanager.md

```diff
@@ -0,0 +1,106 @@
+# The `ThaliReplicationManager` class
```

Comment:
It would probably be good to point out that this is just a hack for story 0 and that this class will be changed and enhanced significantly in the future.